sound/soc/sof/ipc4-priv.h

```diff
@@ -81,6 +82,7 @@ struct sof_ipc4_fw_data {
 	u32 mtrace_log_bytes;
 	int max_num_pipelines;
 	u32 max_libs_count;
+	u32 max_core_frequency;
```

Comment:
@ujfalusi No 4.3+Ghz DSP support? :) Oh well, the IPC4 interface is limited to u32 already, so probably safe to go with this (famous last words)

Reply:
Imagine this trips when we run SOF with Zephyr on a CPU core (such as on i.MX93) and the core somehow shoots up above that (it doesn't for this specific case). Future ARM based boards which do this separate-core thingy where this can be an issue?

Reply:
It is a u32 in the received message, I think a new tuple could be added for representing it in MHz, KHz?